docs/distribution/README.md

```diff
@@ -0,0 +1,528 @@
+      "mediaType": "application/vnd.oci.image.index.v1+json",
+      "size": "1025",
+      "config-mediaType": "application/vnd.cncf.notary.config.v2+jwt"
+    },
+    {
+      "digest": "sha256:007170c33ebc4a74a0a554c86ac2b28ddf3454a5ad9cf90ea8cea9f9e75a153b",
+      "mediaType": "application/vnd.oci.image.index.v1+json",
+      "size": "1025",
+      "config-mediaType": "application/vnd.oci.image.index.v1+json"
+    }
+  ]
+}
+```
+
+## Signature Pull
+
+Using one of the options from the [Signature Discovery](#signature-discovery) section, a specific digest is resolved. A signature shall be pulled using the distribution spec [GET Manifest][distribution-spec-get-manifest] API.
+
+## Example Artifacts
+
+The following are references used in the examples below.
+
+These assume:
+
+* The original net-monitors image was sourced from `registry.wabbit-networks.com/net-monitor:v1`.
+* Wabbit Networks signed the original image
+* ACME Rockets imported the net-monitor image into `registry.acme-rockets.com/net-monitor:v1`
+* The Wabbit Networks signature was copied into `registry.acme-rockets.com/net-monitor:v1`
+* ACME Rockets added a verification signature.
+* Signature objects do NOT have tags. However, they are placed in the same repo as the artifact they reference.
+* Per the design options, a signature object may be persisted as an OCI Manifest or OCI Index.
+
+|Artifact                 |`config.mediaType`                         | Digest                                                                  |
+|-------------------------|-------------------------------------------|-------------------------------------------------------------------------|
+|`net-monitor:v1` image   |`application/vnd.oci.image.config.v1+json` |`sha256:2235d2d22ae5ef400769fa51c84717264cd1520ac8d93dc071374c1be49cc77c`|
+|wabbit-networks signature|`application/vnd.cncf.notary.config.v2+jwt`|`sha256:90659bf80b44ce6be8234e6ff90a1ac34acbeb826903b02cfa0da11c82cbc042`|
+|acme-rockets signature   |`application/vnd.cncf.notary.config.v2+jwt`|`sha256:007170c33ebc4a74a0a554c86ac2b28ddf3454a5ad9cf90ea8cea9f9e75a153b`|
+
+### Example manifest for the **container image**: `registry.acme-rockets.com/net-monitor:v1`
+
+```json
+{
+  "schemaVersion": 2,
+  "mediaType": "application/vnd.oci.image.manifest.v2+json",
+  "config": {
+    "mediaType": "application/vnd.oci.image.config.v1+json",
+    "digest": "sha256:2235d2d22ae5ef400769fa51c84717264cd1520ac8d93dc071374c1be49cc77c",
+    "size": 1906
+  },
+  "layers": [
+    {
+      "mediaType": "application/vnd.oci.image.layer.v1.tar+gzip",
+      "digest": "sha256:9834876dcfb05cb167a5c24953eba58c4ac89b1adf57f28f2f9d09af107ee8f0",
+      "size": 32654
+    },
+    {
+      "mediaType": "application/vnd.oci.image.layer.v1.tar+gzip",
+      "digest": "sha256:ec4b8955958665577945c89419d1af06b5f7636b4ac3da7f12184802ad867736",
+      "size": 73109
+    }
+  ]
+}
+```
+
+### Example **config object** for the **Notary v2 signature artifact**
```

Comment:
Is this the signature provided by Wabbit Networks or the additional signature added by ACME Rockets?  (I'm asking this because the "references" list only shows the ACME Rockets name, which implies either that this is the signature generated by ACME Rockets or that ACME Rockets mutated the Wabbit Networks signature manifest when importing the image into their registry.)

Reply:
see latest updates to hopefully provide more clarity on which is which